shellmate-prd.md

```
@@ -4,6 +4,25 @@
 **Date** : 2026-02-15
 **Statut** : En cours de développement
 
+### Table des matières
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)